Source/Core/Common/SymbolDB.h

```diff
@@ -35,18 +35,25 @@ struct Symbol
 
   void Rename(const std::string& symbol_name);
 
-  std::string name;
-  std::string function_name;   // stripped function name
-  std::vector<SCall> callers;  // addresses of functions that call this function
-  std::vector<SCall> calls;    // addresses of functions that are called by this function
-  u32 hash = 0;                // use for HLE function finding
-  u32 address = 0;
-  u32 flags = 0;
-  u32 size = 0;
-  int num_calls = 0;
-  Type type = Type::Function;
-  int index = 0;  // only used for coloring the disasm view
-  bool analyzed = false;
+  std::string m_name;
```

Comment:
The `m_` prefix should be used on `class` members not `struct`. Moreover, I'm strongly opposed to the modification of this struct if there isn't any real benefit from it such as adding fields like `file_offset` or `virtual_address`. These are CodeWarriors specific, you're also missing the object file and the linked/library file for said CW symbol.

Reply:
I was not aware of the distinction between classes and structs before.  I reverted the name changes.
I disagree about the CW specific stuff being bad, but I don't have to agree.

I just have to get this PR accepted.